packages/ui/origin-ui-core/src/features/certificates/types.ts

```diff
@@ -15,7 +16,7 @@ export interface ICertificate {
     energy: IEnergy;
     isOwned: boolean;
     isClaimed: boolean;
-    myClaims?: any[];
+    myClaims?: ClaimDTO[];
```

Comment:
I don't think we need to have this type `ICertificate` defined in the UI packages. 

It's better to use the `CertificateDTO` class from `@energyweb/issuer-api-client`, so that we don't have to update these UI classes every time, but they are automatically updated through the client.

Reply:
We used to define those classes manually because we didn't have the automatically generated definitions from the client, and we didn't want to import the backend packages into the UI package.

So I consider these "copied" types to be legacy.

Reply:
I totally agree with, but I suggest making this switch as a separate PR, because there are a lot of places where we should change it e.g. exchange-ui-core